ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly-4.14-upgrade-from-stable-4.14.yaml

```diff
@@ -280,6 +584,27 @@ tests:
     - ref: ipi-install-heterogeneous
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-gcp-ipi-ovn-ipsec
+- as: gcp-ipi-proxy-etcd-encryption-arm-f28
```

Comment:
We have other jobs covered `proxy` and `etcd-encryption`, so we can remove this one.